test/osbuildtest.py

```diff
@@ -15,21 +15,102 @@ class TestCase(unittest.TestCase):
     store is used, which can be accessed through `self.store`.
     """
 
-    def setUp(self):
-        self.store = tempfile.mkdtemp(dir="/var/tmp")
+    @classmethod
+    def setUpClass(cls):
+        cls.store = tempfile.mkdtemp(dir="/var/tmp")
 
-    def tearDown(self):
-        shutil.rmtree(self.store)
+    @classmethod
+    def tearDownClass(cls):
+        shutil.rmtree(cls.store)
```

Comment:
I'm doing the same for testing image-info, but I think we need to be a bit careful here, where we actually test osbuild itself. Tests should be as independent from each other as possible.

I'm not entirely against it. If we go this way, please include a comment pointing this out as the doc string for this test class.